src/endpoint/schema.ts

```diff
@@ -259,8 +259,11 @@ export class SchemaEndpoint extends Endpoint {
 	/**
 	 * Returns a list of all ST Schema C2C connectors belonging to the principal (i.e. the user)
 	 */
-	public async list(): Promise<SchemaApp[]> {
-		const response = await this.client.get<SchemaAppList>('apps')
+	public async list(options?: { includeAllOrganizations?: boolean }): Promise<SchemaApp[]> {
```

Comment:
We have been doing this style of arguments more and more in the CLI but haven't much yet in the core SDK so it's maybe a little out-of-place for now but you have to start somewhere and calling code reads so much better when with it.